tests/orquestra/quantum/operators/_operators_test.py

```diff
@@ -641,3 +641,14 @@ def test_paulisum_repr(self, pauli_sum):
     def test_paulisum_multiply_by_zero(self, pauli_sum: PauliSum):
         zero_op = pauli_sum * 0
         assert zero_op == PauliSum()
+
+    def test_paulisum_constant_value_zero(self, pauli_sum):
+        assert 0 == pauli_sum.constant_value
```

Comment:
[nitpick] Usually we do asserts the other way:
`assert pauli_sum.constant_value==0`

Reply:
Done